openhtf/util/measurements.py

```diff
@@ -151,15 +151,15 @@ def WithArgs(self, **kwargs):
     new_meas.docstring = formatter(self.docstring)
     return new_meas
 
-  def __getattr__(self, attr):
+  def __getattr__(self, attr):  # pylint: disable=invalid-name
```

Comment:
agreed, but that's a separate issue that's already been filed and should be done in a separate PR :)